lib/client/ipam_block_reader_writer.go

```diff
@@ -139,6 +124,20 @@ func (rw blockReaderWriter) claimNewAffineBlock(
 	return nil, noFreeBlocksError("No Free Blocks")
 }
 
+// isIPPoolInRequestedPools checks if the IP Pool that is passed in belongs to the list of IP Pools
+// that should be used for assigning IPs from.
+func isIPPoolInRequestedPools(pool net.IPNet, requestedPools []cnet.IPNet) bool {
```

Comment:
@robbrockbank we mention it here: http://docs.projectcalico.org/master/reference/cni-plugin/configuration#ipam 
```
By default, Calico IPAM will assign IP addresses from all the available IP Pools.

Optionally, the list of possible IPv4 and IPv6 pools can also be specified via the following properties:
```